### PR TITLE
tests: using systemd-run instead of manually create a systemd unit - part 1

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This is a task to start replacing the system helper by systemd-run for non-persistent services.

### CODE REVIEW

File: tests/core/custom-device-reg-extras/task.yaml [hunk 1]
```diff
@@ -37,7 +35,8 @@ prepare: |
     prepare_testrootorg_store
 
     # start fake device svc
-    systemd_create_and_start_unit fakedevicesvc "$(command -v fakedevicesvc) localhost:11029"
```
Comment (mvo5) [inline tests/core/custom-device-reg-extras/task.yaml:40]: Can we remove this helper "systemd_create_and_start_unit " now or is it still used?

**Review by mvo5 — APPROVED** [review]
Looks good, thanks for this cleanup!

**Comment by mvo5** [comment]
There are some shellcheck errors though that need addressing.

File: tests/core/custom-device-reg-extras/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/core/custom-device-reg-extras/task.yaml:40]: It is still used, there will be a second part for the migration and then we will be able to remove it

**Pull request merged**